server/ServerFSM.cpp

```diff
@@ -1188,7 +1194,41 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
             // limit count of AI
             auto max_ai = GetOptionsDB().Get<int>("network.server.ai.max");
             if (ai_count <= max_ai || max_ai < 0) {
-                m_lobby_data->m_players    = incoming_lobby_data.m_players;
+                if (sender->HasAuthRole(Networking::ROLE_HOST)) {
+                    // don't check host.
+                    m_lobby_data->m_players    = incoming_lobby_data.m_players;
+                } else {
+                    // check player doesn't set protected empire to yourself
```

Comment:
yourself -> themselves ?

not sure what this means...

Reply:
Fixed. It means player could set protected empire only if player have requested name.